### DISCUSSION TITLE
Everything scanned in, ASN's at the ready...

### DISCUSSION THREAD

**Discussion by RefineryX** [jonaswinkler/paperless-ng#1554] [General]
... these small labels are perfect from Amazon!

![IMG_0276](https://user-images.githubusercontent.com/46373533/149789910-8e148799-f637-4f62-8972-b321d035a6cd.jpeg)

https://www.amazon.co.uk/Avery-L4731REV-25-Self-Adhesive-Removable-Labels/dp/B000J69UJM/ref=sr_1_1?crid=2GMG52J1XW0MX&keywords=avery+189+removable&qid=1641982382&sprefix=avery+189+removeable%2Caps%2C76&sr=8-1

**Comment by TylonHH** [comment]
Or [what about this](https://amzn.to/3QzxhAM)? It's more fancy

**Reply by RefineryX** [reply]
404
